src/pandapipes/pf/pipeflow_setup.py

```diff
@@ -775,6 +775,8 @@ def reduce_pit(net, mode="hydraulics"):
 
 def check_infeed_number(node_pit):
     slack_nodes = node_pit[:, NODE_TYPE_T] == T
+    if len(node_pit) == 1:
+        node_pit[slack_nodes, INFEED] = True
     infeed_nodes = node_pit[:, INFEED]
     if sum(infeed_nodes) != sum(slack_nodes):
```

Comment:
can we use np.sum here? Since this is related to the whole node_pit, the sum could be very inefficient for large networks.

Reply:
Done